dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -3,9 +3,19 @@
 mkdir -p /etc/frr
 mkdir -p /etc/supervisor/conf.d
 
-sonic-cfggen -d -t /usr/share/sonic/templates/supervisord/supervisord.conf.j2 > /etc/supervisor/conf.d/supervisord.conf
-
-CONFIG_TYPE=`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["docker_routing_config_mode"]'`
+CFGGEN_PARAMS=" \
+    -d \
+    -y /etc/sonic/constants.yml \
+    -t /usr/share/sonic/templates/supervisord/frr_vars.j2 \
```

Comment:
Why do we need frr_vars?
Also why you have only source after -t?

Reply:
the frr_vars is a template file that pulls information about the config_type. This variable later decides if BGP service is integrate into vtysh or not. The content of the frr_vars is retreiving single variable for now:
`{{ DEVICE_METADATA["localhost"]["docker_routing_config_mode"] }}`
Lack of destination after the first argument, means this template is going to be rendered to stdout.

In comparison, commit [a5f2de](https://github.com/Azure/sonic-buildimage/pull/5176/commits/a5f2de02edb06bf2c8e34103753b1525258c672e) does docker initialization with two calls to cfggen. The time difference between the two approaches is about 1sec.